mkdocs.yml

```diff
@@ -5,6 +5,9 @@ site_dir: site
 docs_dir: site-src
```

Comment:
Recommend adding a link to the new implementation-tables page somewhere in the navigation so it doesn't feel quite as disconnected from the rest of the site.

Maybe we have an "Implementations" section with "List" (current page) and "Comparison" (new page) options.